src/CoreBundle/Resources/config/mappers.xml

```diff
@@ -22,6 +22,8 @@
         </service>
 
         <service id="chameleon_system_core.mapper.media_field" class="TCMSMediaFieldMapper">
+            <argument type="service" id="chameleon_system_core.media_manager.url_generator" />
```

Comment:
Those lines are no longer required due to the new decorator.

Reply:
reverted the change.